pkg/proctree/process.go

```diff
@@ -22,22 +22,15 @@ type Process struct {
 	mutex *sync.RWMutex // mutex to protect the process
 }
 
-const (
-	executableChangelogSize = 5 // Binary's history is much more important to save
-	// TODO: Decide whether remove the interpreter and interp from the tree or add them back
-	interpreterChangelogSize = 0
-	interpChangelogSize      = 0
-)
-
 // NewProcess creates a new process.
 func NewProcess(hash uint32) *Process {
 	return &Process{
 		processHash: hash,
 		parentHash:  0,
 		info:        NewTaskInfo(),
-		executable:  NewFileInfo(executableChangelogSize),
-		interpreter: NewFileInfo(interpreterChangelogSize),
-		interp:      NewFileInfo(interpChangelogSize),
+		executable:  NewFileInfo(),
+		interpreter: NewFileInfo(),
+		interp:      NewFileInfo(),
```

Comment:
This also turns interpreters fields on, previously disabled by: https://github.com/aquasecurity/tracee/pull/4058

So, the heap reduction already considers the interpreter fields enabled again.

Reply:
I think we can remove these two fields for now - this should save a few extra bytes.
We can add them in the future when required.

Reply:
Done, we got extra 8MB with 32k cache size (both): https://github.com/aquasecurity/tracee/pull/4384/commits/8f80d65132b0fba11c60dbcac5685efbcc047e4b